### ISSUE TITLE
A malformed JWT token should provide a 400 bad request

### ISSUE THREAD

**Issue by gregtatum** [firefox-devtools/profiler-server#51]
I am getting a 500 Internal Server Error, when really it is an error from the client side, which should be 4xx. I think 400 fits the bill for this particular error.

Here is the error from the server:

```
FirefoxProfiler.app.ERROR: server_error {"error":"JsonWebTokenError: jwt malformed","stack":"JsonWebTokenError: jwt malformed\n    at Object.module.exports [as verify] (/Users/greg/dev/server-profiler/node_modules/jsonwebtoken/verify.js:63:17)\n    at verify (/Users/greg/dev/server-profiler/src/logic/jwt.js:44:16)\n    at /Users/greg/dev/server-profiler/src/routes/delete-profile.js:40:21\n    at dispatch (/Users/greg/dev/server-profiler/node_modules/koa-compose/index.js:44:32)\n    at next (/Users/greg/dev/server-profiler/node_modules/koa-compose/index.js:45:18)\n    at /Users/greg/dev/server-profiler/node_modules/koa-json-body/lib/index.js:25:14\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at /Users/greg/dev/server-profiler/src/middlewares/reportTo.js:52:5\n    at logger (/Users/greg/dev/server-profiler/node_modules/koa-logger/index.js:67:7)"}
```

Here was my curl:

```
➤ curl \
>   --include \
>   --header 'Accept: application/vnd.firefox-profiler+json;version=1.0' \
>   --request POST \
>   --data '{"jwtToken":"________PUT_JWT_TOKEN_HERE_______"}' \
>   http://localhost:5252/delete-profile
HTTP/1.1 500 Internal Server Error
Content-Type: text/plain; charset=utf-8
Content-Length: 21
Date: Wed, 25 Mar 2020 21:42:12 GMT
Connection: keep-alive

Internal Server Error
```

**Comment by julienw** [comment]
Well, this is part of your new code, why don't you fix it in #52 directly ? :-)

**Comment by gregtatum** [comment]
Yeah, I didn't realize I should be catching the error in the route, so it's working as expected. I can catch and handle it as a bad request.
